routers/web/repo/webhook.go

```diff
@@ -99,7 +99,7 @@ func getOwnerRepoCtx(ctx *context.Context) (*ownerRepoCtx, error) {
 			IsAdmin:         true,
 			IsSystemWebhook: ctx.Params(":configType") == "system-hooks",
 			Link:            path.Join(setting.AppSubURL, "/admin/hooks"),
-			LinkNew:         path.Join(setting.AppSubURL, "/admin/system-hooks"),
+			LinkNew:         "/admin/" + ctx.Params(":configType"),
```

Comment:
AppSubURL should not be removed.

Reply:
omg sorry, that was dumb of me :face_with_spiral_eyes: . I've fixed it.